src/components/NavBarOverlay/NavBarOverlay.stories.tsx

```diff
@@ -25,39 +26,106 @@ export const Default: Story = {
+        collapsed={collapsed}
+        selected={selectedRoute.startsWith('/metamask')}
+        subLinksCollapsed={mmCollapsed}
+        setSubLinksCollapsed={() => mmSetCollapsed(!mmCollapsed)}
+        subLinks={[
+          <SubLink
+            key={'Portfolio'}
+            onClick={() => setSelectedRoute('/metamask/Portfolio')}
+            selected={selectedRoute === '/metamask/Portfolio'}
+          >
+            Portfolio
+          </SubLink>,
+          <SubLink
+            key={'Swap'}
+            onClick={() => setSelectedRoute('/metamask/Swap')}
+            selected={selectedRoute === '/metamask/Swap'}
+          >
+            Swap
+          </SubLink>,
+          <SubLink
+            key={'Bridge'}
+            onClick={() => setSelectedRoute('/metamask/Bridge')}
+            selected={selectedRoute === '/metamask/Bridge'}
+          >
+            Bridge
+          </SubLink>,
+          <SubLink
+            key={'Buy'}
+            onClick={() => setSelectedRoute('/metamask/Buy')}
+            selected={selectedRoute === '/metamask/Buy'}
+          >
+            Buy
+          </SubLink>,
+          <SubLink
+            key={'Sell'}
+            onClick={() => setSelectedRoute('/metamask/sell')}
+            selected={selectedRoute === '/metamask/sell'}
+          >
+            Sell
+          </SubLink>
+        ]}
       />,
       <NavItem
         title={'Marketplace'}
-        route={'/marketplace'}
         icon={<Home fill="white" />}
         key={'/marketplace'}
-        onClick={() => setSelectedRoute('/marketplace')}
+        onClick={() => marketplaceSetCollapsed(!marketplaceCollapsed)}
         collapsed={collapsed}
-        currentRoute={selectedRoute}
+        selected={selectedRoute.startsWith('/marketplace')}
+        subLinksCollapsed={marketplaceCollapsed}
+        setSubLinksCollapsed={() =>
+          marketplaceSetCollapsed(!marketplaceCollapsed)
+        }
+        subLinks={[
+          <SubLink
+            key={'NTx'}
+            onClick={() => setSelectedRoute('/marketplace/NTx')}
+            selected={selectedRoute === '/marketplace/NTx'}
+          >
+            NTx
+          </SubLink>,
+          <SubLink
+            key={'XocietyFrontier'}
+            onClick={() => setSelectedRoute('/marketplace/XocietyFrontier')}
+            selected={selectedRoute === '/marketplace/XocietyFrontier'}
+          >
+            XocietyFrontier
+          </SubLink>
+        ]}
       />,
       <NavItem
         title={'Achievements'}
-        route={'/achievements'}
         icon={<TrophyOutline fill="white" />}
         key={'/achievements'}
-        onClick={() => setSelectedRoute('/achievements')}
+        onClick={() => console.log('coming soon')}
         collapsed={collapsed}
-        currentRoute={selectedRoute}
         secondaryTag={'Coming Soon'}
         style={{
-          cursor: 'not-allowed'
+          cursor: 'not-allowed',
+          opacity: '50%'
```

Comment:
let's take the opportunity to remove the inline style

Reply:
this is just a story though

Reply:
ahhhh never mind then